README.md

```diff
@@ -3,4 +3,29 @@ cosmoz-bottom-bar
 
 [![Build Status](https://travis-ci.org/Neovici/cosmoz-bottom-bar.svg?branch=master)](https://travis-ci.org/Neovici/cosmoz-bottom-bar)
 
-The Polymer element **cosmoz-bottom-bar** is a responsive bottom-bar that can house buttons/actions and a menu for the buttons that won't fit the available width.
\ No newline at end of file
+## &lt;cosmoz-bottom-bar&gt;
+
+**cosmoz-bottom-bar** is a responsive bottom-bar that can house buttons/actions and a menu for the buttons that won't fit the available width.
+
+Example:
+
+<!---
+```
+<custom-element-demo>
+  <template>
+    <script src="../webcomponentsjs/webcomponents-lite.js"></script>
+    <link rel="import" href="cosmoz-button-bar.html">
+    <div id="container">
+      <cosmoz-bottom-bar style="min-width:200px; max-width: 200px;">
+      		<fixed-size-item width="50px" height="32px"></fixed-size-item>
+      </cosmoz-bottom-bar>
+    </div>
+  </template>
+</custom-element-demo>
+```
+-->
+```html
```

Comment:
There is a typo in the import link: it's not `cosmoz-button-bar` but `cosmoz-bottom-bar`.
I just created issue #7 if you want to add refs to your commits.